src/pages/on-chain-governance/_components/SubmitProposal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { CopyButton } from "@components/commons/CopyButton";
+
+interface SubmitProposalProps {
+  command: string;
+}
+export function SubmitProposal({ command }: SubmitProposalProps): JSX.Element {
+  return (
+    <>
+      <span className="text-gray-600 dark:text-gray-100 text-sm md:text-base">
+        To finalize your submission on-chain, copy the generated command line,
+        and paste it in the CLI located in your full node wallet. If you have
+        not downloaded a full node wallet,
+        <a
+          className="text-[#4A72DA]"
+          href="https://defichain.com/downloads"
+          target="_blank"
+          rel="noreferrer"
+        >
+          &nbsp;download here
+        </a>
+      </span>
+      <div className="flex flex-row mt-6 mb-2 md:mb-4 bg-blue-100 rounded py-3 px-4 items-center">
+        <span className="text-[#4A72DA] break-all">{command}</span>
```

Comment:
```suggestion
        <span className="text-[#4A72DA] break-all flex grow">{command}</span>
```

screen width for 1550px copy icon wouldnt be at the end 
<img width="890" alt="image" src="https://user-images.githubusercontent.com/57183851/208868787-f196b8ab-7074-45df-ac81-a1da3b5c42f2.png">